src/widget/wtracktableview.cpp

```diff
@@ -1929,6 +1973,22 @@ void WTrackTableView::lockBpm(bool lock) {
     }
 }
 
+void WTrackTableView::slotColorPicked(PredefinedColorPointer pColor) {
+    TrackModel* trackModel = getTrackModel();
+    if (trackModel == nullptr) {
+        return;
+    }
+
+    QModelIndexList selectedTrackIndices = selectionModel()->selectedRows();
```

Comment:
`const` or `qAsConsr` to avoid copy-on-write in for-each loop

Reply:
This is missing in all functions of the WTrackTableView. Fixed in 1bca6d0838cbf2217726cd60d81aa6fcb067ba97.